website/content/en/docs/concepts/disruption.md

```diff
@@ -138,13 +138,12 @@ In special cases, drift can correspond to multiple values and must be handled di
 #### Behavioral Fields
 Behavioral Fields are treated as over-arching settings on the NodePool to dictate how Karpenter behaves. These fields don’t correspond to settings on the NodeClaim or instance. They’re set by the user to control Karpenter’s Provisioning and disruption logic. Since these don’t map to a desired state of NodeClaims, __behavioral fields are not considered for Drift__.
 
-__Behavioral Fields__
-- Weight                      
-- Limits                      
-- ConsolidationPolicy               
-- ConsolidateAfter      
-- ExpireAfter  
----      
+##### NodePool
+| Fields         |
+|----------------|
+| Weight         |
+| Limits         |
+| Disruption     | 
```

Comment:
Since Disruption isn't a field, but a set of fields, should we call this out as Disruption Fields?

Reply:
One option is to call out the paths directly e.g. `spec.weight`, `spec.limits`, `spec.disruption.consolidationPolicy`, etc.

Reply:
Yeah, I like calling out the paths. That would be more explicit.